src/utils/mtev_logic.c

```diff
@@ -94,9 +94,11 @@ struct mtev_logic_var_t {
     int64_t     l;
     double      n;
   } value;
+  size_t s_len;
   pcre *re;
   pcre_extra *re_extra;
   mtev_boolean free_s;
+  char internal[40]; /* up 39 byte copies without alloc */
```

Comment:
Wouldn't it be better to define 40 as a constant somewhere, and use it below instead of checking `sizeof(foo.internal)`?  Or does the compiler know that never changes and already replace it with a constant?

Reply:
Compiler knows. And sizeof is our coding style.